bentoml/_internal/cli/bento_server.py

```diff
@@ -15,7 +15,7 @@
 
 def add_serve_command(cli: click.Group) -> None:
     @cli.command()
-    @click.argument("bento", type=click.STRING)
+    @click.argument("bento", type=click.STRING, default=".")
```

Comment:
So this would now let us run: "bentoml serve ."?

We previously didn't allow this. But I like it. Isn't there somewhere that doesn't allow serving a particular bento name based on a regex?

Reply:
I think at some point we had this error: "bentoml.exceptions.InvalidArgument: . is not a valid tag: a tag's name or version must consist of alphanumeric characters, '_', '-', or '.', and must start and end with an alphanumeric character"

Reply:
@timliubentoml yes, both `bentoml serve` and `bentoml serve .` should just work now, as long as there's a `bentofile.yaml` file in the directory